Ladybird/AppKit/Application/ApplicationBridge.cpp

```diff
@@ -37,13 +39,47 @@ ErrorOr<void> ApplicationBridge::launch_request_server(Vector<ByteString> const&
     return {};
 }
 
+static ErrorOr<NonnullRefPtr<ImageDecoderClient::Client>> launch_new_image_decoder()
+{
+    auto image_decoder_paths = TRY(get_paths_for_helper_process("ImageDecoder"sv));
+    return launch_image_decoder_process(image_decoder_paths);
+}
+
+ErrorOr<void> ApplicationBridge::launch_image_decoder()
+{
+    m_impl->image_decoder_client = TRY(launch_new_image_decoder());
```

Comment:
@trflynn89 this duplicated logic looks like a good candidate for LibWebView..?

Not sure how much of the Process management we want to move there vs keeping in the UI arena though 🤔 .

I also want to get rid of the Web::Platform::ImageDecoderPlugin interface, since at the moment it literally serves no purpose, as we don't have an alternative to the Ladybird one (that we care about :P). Especially now that I've moved the process creation responsibility out of the plugin.

Reply:
Yeah, seems like something a `WebView::Application` class could handle. I think we've talked about making something like that before to dedup the logic here.